### PR TITLE
fix: claimed and update polkadot api

### PR DESCRIPTION
**Pull request by rubenkristian**
### JIRA LINK 🟦
- https://blocksphere2020.atlassian.net/browse/DBIO-2727

### Changelog/Description 🗒️
- update polkadot api
- fix country service request delete

### WARNING ⚠️
- 

### Screenshoot/Video (Optional) 🖼️
image

### CODE REVIEW

**Review by agustinustheo — CHANGES_REQUESTED** [review]
Please make integration testing for this.

File: src/substrate/events/service-request/commands/claimed-service-request/claimed-service-request.handler.ts [hunk 1]
```diff
@@ -44,11 +44,12 @@ export class ClaimedServiceRequestHandler
 
     const service_request = body.hits?.hits[0]?._source || null;
 
-    if (service_request != null) {
+    if (service_request !== null) {
```
Comment (bukandicki) [inline src/substrate/events/service-request/commands/claimed-service-request/claimed-service-request.handler.ts:47]: too many codes inside this `if` statement, please make it simpler by using the return instead

File: src/substrate/events/service-request/commands/finalized-service-request/finalized-service-request.handler.ts [hunk 2]
```diff
@@ -44,11 +44,12 @@ export class FinalizedServiceRequestHandler
 
     const service_request = body.hits?.hits[0]?._source || null;
 
-    if (service_request != null) {
+    if (service_request !== null) {
```
Comment (bukandicki) [inline src/substrate/events/service-request/commands/finalized-service-request/finalized-service-request.handler.ts:47]: Same here too many codes inside this `if` statement, please make it simpler by using the return instead

File: src/substrate/events/service-request/commands/processed-service-request/processed-service-request.handler.ts [hunk 3]
```diff
@@ -44,11 +44,12 @@ export class ProcessedServiceRequestHandler
 
     const service_request = body.hits?.hits[0]?._source || null;
 
-    if (service_request != null) {
+    if (service_request !== null) {
```
Comment (bukandicki) [inline src/substrate/events/service-request/commands/processed-service-request/processed-service-request.handler.ts:47]: too many codes inside this `if` statement, please make it simpler by using the return instead

File: src/substrate/substrate.handler.ts [hunk 4]
```diff
@@ -458,6 +470,7 @@ export class SubstrateService
     if (this.listenStatus) {
       this.logger.log('Restart connection to node.');
       this.stopListen();
+      await this.setWsProvider();
       await this.startListen();
     }
```
Comment (bukandicki) [inline src/substrate/substrate.handler.ts]: too many codes inside this `if` statement, please make it simpler by using the return instead

**Review by bukandicki — DISMISSED** [review]
Looks good, but i have a note for code quality

**Review by agustinustheo — APPROVED** [review]

**Comment by bukandicki** [comment]
@rubenkristian I mean, instead of
```js
if(service_request !== null) {  ...a bunch of code }
```
make it simpler like this, not convert those code to a function
```js
if(service_request === null) return

// Move here please
 ... a bunch of code
```

**Comment by agustinustheo** [comment]
> @rubenkristian I mean, instead of
> 
> ```js
> if(service_request !== null) {  ...a bunch of code }
> ```
> 
> make it simpler like this, not convert those code to a function
> 
> ```js
> if(service_request === null) return
> 
> // Move here please
>  ... a bunch of code
> ```

After pulling and reading the code, I see that this is fine and is more readable than your suggestion.

**Pull request merged**